lib/tapioca/commands/abstract_gem.rb

```diff
@@ -252,6 +262,26 @@ def expected_rbis
           .to_h { |gem| [gem.name, gem.version.to_s] }
       end
 
+      sig do
+        params(
+          gem_name: String,
+          resolved_dependencies: T::Array[Tapioca::Gemfile::GemSpec],
+        ).returns(T::Array[Tapioca::Gemfile::GemSpec])
+      end
+      def gem_with_dependencies(gem_name, resolved_dependencies = [])
+        gem = @bundle.gem(gem_name)
+        return [] if gem.nil?
```

Comment:
I think we want to return the `gem_name` in this case so that we can raise on line 84.